iot-hub/Samples/service/AutomaticDeviceManagementSample/Parameters.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using System;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments.
+    /// If the parameter is not supplied via command line args, it will look for it in environment variables.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "HubConnectionString",
+            HelpText = "The IoT Hub connection string. This is available under the \"Shared access policies\" in the Azure portal." +
```

Comment:
Guidance we have from the docs team is that **hub** is not capitalized.

```suggestion
            HelpText = "The IoT hub connection string. This is available under the \"Shared access policies\" in the Azure portal." +
```